docs/docs/hooks.md

```diff
@@ -193,6 +193,27 @@ Model.beforeBulkDestroy(function(whereClause, fn) {
 Model.destroy({ where: {username: 'Tom'}} /*whereClause argument*/)
 ```
 
+If you use `Model.bulkCreate(...)` with the `updatesOnDuplicate` option, changes made in the hook to fields that aren't given in the `updatesOnDuplicate` array will not be persisted to the database. However it is possible to change the updatesOnDuplicate option inside the hook if this is what you want.
+
+```
+// Bulk updating existing users with updatesOnDuplicate option
+Users.bulkCreate({ id: 1, isMemeber: true}, 
```

Comment:
Whoops, good spotting. Fixed in https://github.com/ashokfernandez/sequelize/commit/12acc40cac2c8ca8e4a73b514056aa632d0f1345.